cmd/openshift-tests/openshift-tests.go

```diff
@@ -169,6 +174,9 @@ func newRunUpgradeCommand() *cobra.Command {
 				if len(upgradeOpt.ToImage) == 0 {
 					return fmt.Errorf("--to-image must be specified to run an upgrade test")
 				}
+				if err := initCSITests(); err != nil {
```

Comment:
Move this into initProvider, unless there is a reason for it to be separate.

Reply:
moved